hs_core/views/__init__.py

```diff
@@ -41,6 +41,7 @@
 from . import resource_rest_api
 from . import user_rest_api
 from . import resource_folder_hierarchy
+from . import resource_folder_api
```

Comment:
@alvacouch May consider to change ```resource_folder_api``` module to ```resource_folder_rest_api``` to be consistent with existing naming such as ```resource_rest_api```, ```user_rest_api```, etc.